rllib/evaluation/rollout_worker.py

```diff
@@ -714,6 +720,24 @@ def wrap(env):
                     f"[{method.__name__}]",
                     error=False,
                 )
+            elif method == "dm":
```

Comment:
Actually, I think these shorts are not necessary anymore. We only ever supported "is" and "wis" and now that we have the new API, we should just force users to specify a class here.

Reply:
They're necessary for the command-line API e.g. ```rllib train```, which is currently used in the [Offline RL docs](https://docs.ray.io/en/latest/rllib/rllib-offline.html), but I can definitely remove them instead.

Reply:
should we create a simple tune.registry for this? that would look pretty nice.
python object in config sometimes has downside, like I hope we don't have any python obj values in our policy_config ...